jwst/pipeline/calwebb_spec2.py

```diff
@@ -386,11 +386,17 @@ def process_exposure_product(
             resampled = self.resample_spec.run(resampled)
 
         elif (exp_type in ["MIR_MRS", "NRS_IFU"]) or is_nrs_ifu_linelamp(calibrated):
-            # First call pixel_replace then call cube_build step for IFU data.
-            # For cube_build always create a single cube containing multiple
-            # wavelength bands
+            # set the default output type for both instruments if is not set
+            if exp_type == "NRS_IFU" and self.cube_build.output_type is None:
+                self.cube_build.output_type = "band"
+
+            if is_nrs_ifu_linelamp(calibrated):
+                self.cube_build.output_type = "band"
```

Comment:
I think we should treat linelamp cases the same as regular IFU cases - only override if output_type is None.
```suggestion
            if (exp_type == "NRS_IFU" or is_nrs_ifu_linelamp(calibrated)) and self.cube_build.output_type is None:
                self.cube_build.output_type = "band"
```

Reply:
had some trouble with syntax making it one line so I broke it into 2 cases both needing self.cube_build.output_type is None